modules/instance_template/main.tf

```diff
@@ -140,4 +141,12 @@ resource "google_compute_instance_template" "tpl" {
   confidential_instance_config {
     enable_confidential_compute = var.enable_confidential_vm
   }
+
+  dynamic "guest_accelerator" {
+    for_each = local.gpu_enabled ? list(var.gpu) : []
```

Comment:
`list()` is being [deprecated](https://www.terraform.io/docs/language/functions/list.html), we can use `[]` directly

Reply:
Fixed in https://github.com/terraform-google-modules/terraform-google-vm/pull/160/commits/9f7bce0fc23bc8bcd49cfd27320d8c4f8db251d4